src/test/kotlin/no/nav/dagpenger/regel/api/models/SubsumsjonTest.kt

```diff
@@ -12,25 +12,23 @@ internal class SubsumsjonTest {
     @Test
     fun `Map to JSON string`() {
         Subsumsjon(
```

Comment:
Denne er vel også en kandidat for PropertyTesting. All den tid det vi prøver på er å teste roundtrips?

Reply:
Ja, dette er en kandidat, tenkte på det og glemte det